src/Hypergrid.js

```diff
@@ -1257,7 +1258,7 @@ var Hypergrid = Base.extend('Hypergrid', {
 
                 // target is to right of scrollable columns; positive delta scrolls right
                 // Note: The +1 forces right-most column to scroll left (just in case it was only partially in view)
-                (delta = c - dw.origin.x + 1) > 0
```

Comment:
my initial concern is how this affects,  https://github.com/openfin/fin-hypergrid/issues/515. Its a uses case we have for one of our customers